spec/request/session/delete_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe 'POST #delete sign_out', type: :request do
+  let!(:user_attrs) { { user: attributes_for(:user) } }
+  let!(:user) { create(:user, :confirmed) }
+  let!(:auth_headers) { auth_user_headers }
+  subject do
+    delete destroy_api_v1_user_session_path, headers: auth_headers, as: :json
+  end
+
+  context 'when valid' do
+    context 'when signed in' do
```

Comment:
There is no context for `when not signed in`?

Reply:
there is further on, inside the invalid context